### ISSUE TITLE
Not on maven central

### ISSUE THREAD

**Issue by cinterloper** [atomix/atomix-vertx#4]
I see the other atomix components but atomix-vertx comes up blank in a search
http://search.maven.org/#search|ga|1|atomix-vertx

**Comment by kuujo** [comment]
You're right. atomix-vertx has not been published to Maven Central simply because of an abundance of caution wrt the Travis tests. Tests pass consistently for me locally, but for some reason get tons of blocked thread exceptions in Travis. But in willing to push a release. We'll see if anyone can track that down.
